modules/n1ql/pages/n1ql-language-reference/flex-indexes.adoc

```diff
@@ -42,17 +51,14 @@ To understand how a N1QL query can make use of a full-text index, it's important
 Full-text indexes have different semantics to N1QL queries.
 Some of the main differences are described here.
 
-* N1QL field names can have special characters.
-Special characters can be escaped with backticks.
+* N1QL field names can have special characters, which can be escaped with backticks.
+The Search service cannot index fields whose names contain special characters.
```

Comment:
This is not true.

Search service CAN index fields whose names contain special characters, and these fields can be searched by N1QL queries that have escaped field names with special characters with back ticks.

Example doc:
```
{
  "@#$%": "hell"
}
```
FTS Index definition:
```
...
      "default_mapping": {
        "default_analyzer": "keyword",
        "dynamic": false,
        "enabled": true,
        "properties": {
          "@#$%": {
            "dynamic": false,
            "enabled": true,
            "fields": [
              {
                "index": true,
                "name": "@#$%",
                "type": "text"
              }
            ]
          }
        }
      },
...
```

Example query:
```
SELECT META().id
FROM `temp` AS t
USE INDEX(USING FTS)
WHERE `@#$%` = "hell";
```

Reply:
Removed this bullet point.